chia/rpc/rpc_server.py

```diff
@@ -58,12 +67,41 @@ def create(
         ssl_client_context = ssl_context_for_client(ca_cert_path, ca_key_path, crt_path, key_path, log=log)
         return cls(rpc_api, stop_cb, service_name, ssl_context, ssl_client_context)
 
-    async def stop(self) -> None:
+    async def start(self, root_path: Path, self_hostname: str, rpc_port: int, max_request_body_size: int) -> None:
+        if self.environment is not None:
+            raise RuntimeError("RpcServer already started")
+
+        app = web.Application(client_max_size=max_request_body_size)
+        runner = web.AppRunner(app, access_log=None)
+
+        runner.app.add_routes([web.post(route, wrap_http_handler(func)) for (route, func) in self.get_routes().items()])
+        await runner.setup()
+        site = web.TCPSite(runner, self_hostname, int(rpc_port), ssl_context=self.ssl_context)
+        await site.start()
+
+        #
+        # On a dual-stack system, we want to get the (first) IPv4 port unless
+        # prefer_ipv6 is set in which case we use the IPv6 port
+        #
+        if rpc_port == 0:
+            rpc_port = select_port(root_path, runner.addresses)
+
+        self.environment = RpcEnvironment(runner, site, uint16(rpc_port))
+
+    def stop(self) -> None:
```

Comment:
Should this be called close instead of stop? Goes with `close` and `await_closed`

Reply:
Makes sense, changed in [021f081](https://github.com/Chia-Network/chia-blockchain/pull/12768/commits/021f08174659750538675767d1b11cf991941fc9)!